### PR TITLE
Adds composition lesson translation to spanish

### PR DESCRIPTION
**Pull request by setzer777**
Title's very straight :shipit:

### CODE REVIEW

**Comment by doomspork** [comment]
Wow @setzer777, you're fast!  Could @erickgnavar or @jesusangelm take a peek at this?

File: es/lessons/basics/composition.md [hunk 1]
```diff
@@ -0,0 +1,239 @@
...
+- [Módulos](#modules)
+  - [Atributos de módulo](#module-attributes)
+- [Estructuras](#structs)
+- [Composición](#composition)
+  - [`alias`](#alias)
+  - [`import`](#import)
+  - [`require`](#require)
+  - [`use`](#use)
+
+## Modules
```
Comment (erickgnavar) [inline es/lessons/basics/composition.md]: Falta traducir el título

File: es/lessons/basics/composition.md [hunk 2]
```diff
@@ -0,0 +1,239 @@
...
+
++ `moduledoc` — Documenta el módulo actual.
++ `doc` — Documentación para funciones y macros.
++ `behaviour` — Usa OTP o comportamiento definido por el usuario.
+
+## Estructuras
+
+Las estructuras son mapas especiales con un conjunto definido de llaves y valores default. Deben ser definidas dentro de un módulo, y tomarán su nombre. Es común que una estructura sea definida únicamente dentro de un módulo.
+
+Para definir una estructura utilizamos `defstruct` junto con una lista de llaves y valores default:
```
Comment (erickgnavar) [inline es/lessons/basics/composition.md]: `default` -> `valores por defecto`

File: es/lessons/basics/composition.md [hunk 3]
```diff
@@ -0,0 +1,239 @@
...
+```
+
+## Composición
+
+Ahora que sabemos cómo crear módulos y estrucutras, aprendamos cómo incluir funcionalidad existente dentro de ellos con composición.
+Elixir nos provee una variedad de diferentes formas para interactuar con otros módulos, démosle un vistazo a lo que tenemos disponible:
+
+### `alias`
+
+Nos permite darle un alias a los módulos, y son usados frecuentemente en Elixir.
```
Comment (erickgnavar) [inline es/lessons/basics/composition.md]: y son usados -> **que** son usados

File: es/lessons/basics/composition.md [hunk 4]
```diff
@@ -0,0 +1,239 @@
...
+En adición a los pares nombre/aridad, hay dos atomos especiales, `:functions` y `:macros`, las cuales importan únicamente funciones y macros, respectivamente:
+
+```elixir
+import List, only: :functions
+import List, only: :macros
+```
+
+### `require`
+
+Aunque `require/2` es usado menos frecuente, es bastante importante. Haciendo `require` de un módulo asegura que está compilado y cargado. Esto es muy útil cuando necesitamos acceso a las macros de un módulo:
```
Comment (erickgnavar) [inline es/lessons/basics/composition.md]: es usado menos frecuente -> no es usado frecuentemente

**Comment by erickgnavar** [comment]
Hi, @setzer777 I added some notes to the commit.
Thanks for your work

**Comment by setzer777** [comment]
Hi @erickgnavar, thanks for the review, really helpful c: working on the comments

**Comment by setzer777** [comment]
Done haha :shipit:

**Comment by doomspork** [comment]
Awesome!  Thank you @setzer777 and @erickgnavar, I appreciate both your help :grinning:

**Comment by doomspork** [comment]
@setzer777 what is your twitter handle btw?  I like to give credit when I tweet about new lessons/translations.

Thanks again

**Comment by setzer777** [comment]
@doomspork and to everyone else, you can find me on twitter as [@AlbertooRomeroo](https://twitter.com/AlbertooRomeroo).

I was thinking you should be talking more about String in Elixir, from how do they work, simple operations and manipulation, how are they handled inside the BEAM, etc., if you need some help with that, please let me know!

Another suggestion is to add some exercises at the end of the modules, so ElixirSchool can turn into a real school where people's evaluating student's work c: I'd also be glad to help you on that.

I'm about to start using Elixir in production in a few months in my organization, and I've found Elixir super interesting thanks to the tremendous power of the BEAM and the beautiful functional-programming paradigm and the beauty of Ruby. Keep in touch guys!

**Pull request merged**